NEWS.md

```diff
@@ -30,6 +30,8 @@ rticles 0.16
 
 - The minimal version of **knitr** required is 1.30 now.
 
+- `arxiv_article()` now supports header-include and bibstyle. (thanks, @eliocamp)
+
```

Comment:
Can you put this bullet at the top ? 

This will be included in version 0.18, the next CRAN version - not 0.16. 

Thank you !

Reply:
Sorry!